lib/base64url.js

```diff
@@ -1,5 +1,5 @@
 /*!
  * Copyright (c) 2019-2020 Digital Bazaar, Inc. All rights reserved.
  */
-import {encode} from 'base64url-universal';
-export default {encode};
+import {encode, decode} from 'base64url-universal';
+export default {encode, decode};
```

Comment:
Technically don't need `decode` anymore. Also, we don't need this extra file, just import directly.